src/libexpr/primops.cc

```diff
@@ -254,9 +254,14 @@ static RegisterPrimOp primop_import({
     .args = {"path"},
     // TODO turn "normal path values" into link below
     .doc = R"(
-      Load, parse and return the Nix expression in the file *path*. If
-      *path* is a directory, the file ` default.nix ` in that directory
-      is loaded. Evaluation aborts if the file doesn’t exist or contains
+      Load, parse and return the Nix expression in the file *path*.
+
+      The value *path* is converted to a string as described in `builtins.toString`.
```

Comment:
This is not exactly true. E.g. `builtins.toString 123` returns `"123"`, but `import 123` fails (it doesn't try to load the file named `123` but says "cannot coerce an integer to a string").

Reply:
Ah,I missed the `coerceMore` argument to `coerceToString` as called by `coerceToPath`. Thanks.